ubidots/ubidots_in.html

```diff
@@ -1,111 +1,608 @@
+
+
+      <ul>
+          <li>
+            <p>
+              <b> Account Type: </b>Defaults to "Ubidots" (standard account). If you have an education account choose "Ubidots for Education".
+            </p>
+          </li>
+          <li>
+
+            <p>
+              <b> Name: </b> Label of node in Node-Red workspace. If empty, defaults to :"Ubidots in".
+            </p>
+          </li>
+          <li>
+            <p>
+              <b> Token: </b> (required) - Token necessary to authenticate the connection to your account. To obtain your token, login on <a href="https://www.ubidots.com" target:"_blank">ubidots.com</a>, under "My Profile" click on "API Credentials". <a href="https://help.ubidots.com/en/articles/590078-find-your-token-from-your-ubidots-account" target="_blank">Tutorial</a>
+            </p>
+          </li>
+          <li>
+            <p>
+              <b>Device label</b> - The label of the Device to which the node subscribes. Please keep in mind that this field is neglected in case custom topics are used.
+            </p>
+          </li>
+          <li>
+            <p>
+              <b>SSL: </b> By default, all data is sent encrypted via TLS. Uncheck if data should be sent unencrypted.
+            </p>
+          </li>
+          <li>
+            <p>
+              <b>Use Custom Topics: </b> Allows to specify up to 10 custom topics to which the node subscribes to. The topic starts after <code>/v1.6/devices/</code>. <p>A topic has to be in the following format: <code>Device_Label/Variable_Label</code>. Optionally <code>/lv</code> can be added to subscribe to the Last Value only. Topics support wildcard characters: "+" for single level and "#" for multi level.</p>
+            </p>
+          </li>
+          <li>
+            <p>
+              <b>Variable Label: </b> (required) - The name of the Variable the node subscribes to.
+            </p>  
+          </li>  
+          <li>
+            <p>
+              <b>Last Value: </b> By default, the node subscribes to the Last Value of the variable. Uncheck to subscribe to the complete data point object.
+            </p>
+          </li>
+          <li>
+            <p>
+              <b>Add Variable/Add Topic: </b> Adds an additional variable/topic (up to 10).
+            </p>
+          </li>
+      </ul>
+      <p>The output is a JSON object with the topic as key and the Last Value/data point object as value, e.g.: <pre>{"device_label/variable": {"value": 100, "timestamp": 1583523586668, "context": { "key1": "value1", "key2": "value2"}, "created_at": 1583523586732} </pre></p>
+  </p>
+
+  <p>
+      Refer to the <a href="https://ubidots.com/docs/api/mqtt.html" target="_blank">Ubidots documentation</a> for more information about the functionality of the MQTT broker.
+  </p>
+  <p>-----</p>
+  <p>
+    If you get the error: "TypeError: send is not a function", please run <code>npm update node-red</code> and reload Node-Red.
```

Comment:
You can style the `"TypeError: send is not a funcion"` with a `<pre>` tag and a light red background with red text, trying to emulate the output of the console.